docs/index.rst

```diff
@@ -12,9 +12,12 @@ Augur originated as part of `Nextstrain <https://nextstrain.org>`__, an open-sou
 .. toctree::
    :maxdepth: 2
    :caption: Table of contents
+   :name: mastertoc
+   :titlesonly:
 
    installation
+   cli
+   Augur examples <examples>
    usage
-   upgrading
-   api
+   Development API <api>
```

Comment:
If possible, I think it'd be nice to omit the subpages of API (augur.align, augur.ancestral, etc) from this main TOC and essentially hide them behind the API index page. I think we expect them to be less-frequented pages and would be good not to overwhelm or confuse folks with two references to "augur align".

Reply:
Do you know how to hide all second-level headers in the API `toctree` from the Index `toctree` without hiding them on the API page itself? I have not been able to figure it out.

Reply:
I do not! There may not be a way to do it.

Reply:
I wonder if there's a way to apply a `maxdepth` on the top-level `toctree` to individual toc items instead of the whole thing? Like, could we set `maxdepth: 1` and then bump it up to 2 for `cli` or conversely keep it at 2 and bump it down to 1 for `api`?

Reply:
From what I could find, multiple `maxdepth`s are not currently supported, but there's a suggested workaround created from writing multiple `toctree`s. https://stackoverflow.com/questions/54800095/sphinx-adds-headings-from-markdown-file-to-document-structure
Thoughts?

Reply:
Hmm, might be less ideal than how it currently is. Maybe table this suggestion for now? I guess later we could split the index page into user-facing and developer-facing sections with two toctrees and some other explanatory prose.